net-libs/libwebsockets/libwebsockets-3.2.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A flexible pure-C library for implementing network protocols"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
+	lejp libev libevent libressl libuv peer-limits server-status smtp socks5
+	sqlite3 ssl static-libs threads zip"
+
+REQUIRED_USE="
+	http-proxy? ( client )
+	generic-sessions? ( sqlite3 )
+	generic-sessions? ( smtp )
+	smtp? ( libuv )
+	?? ( libev libevent )
+"
+
+RDEPEND="
+	sys-libs/zlib
+	http-proxy? ( net-libs/libhubbub )
+	caps?       ( sys-libs/libcap )
```

Comment:
You could also drop the stylistic style, and follow skel style instead. Check subslots for these if they get linked during compile.

Reply:
I believe I've got this now.  If depended on package does anything other than SLOT="0" then I simply set this to := ... so I hope that's ok.

Reply:
Not quite... you should check how the binaries and libraries are linked... but generally I do it like that if I'm unsure.